src/components/Footer/Footer.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Footer from '.';
+
+test('footer logo image loads', async () => {
+  const { getByAltText } = await render(<Footer />);
+  const logoImage = getByAltText('Deco Choice');
```

Comment:
Please use the `getByAltText` from the `screen` object. The main benefit of using `screen` is that we can use `screen.debug` function [More info](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)

```jsx
import {render, screen} from '@testing-library/react'
const logoImage = screen.getByAltText('Deco Choice');
```